nodejs/eks/dashboard.ts

```diff
@@ -21,14 +21,8 @@ export interface DashboardOptions {}
 
 export function createDashboard(name: string, args: DashboardOptions, parent: pulumi.ComponentResource, k8sProvider: k8s.Provider) {
     // Deploy the latest version of the k8s dashboard.
-    const dashboardYaml = [
-        path.join(__dirname, "dashboard", "kubernetes-dashboard.yaml"),
-        path.join(__dirname, "dashboard", "heapster.yaml"),
-        path.join(__dirname, "dashboard", "influxdb.yaml"),
-        path.join(__dirname, "dashboard", "heapster-rbac.yaml"),
-    ].map(filePath => fs.readFileSync(filePath).toString());
     const dashboard = new k8s.yaml.ConfigGroup(`${name}-dashboard`, {
-        yaml: dashboardYaml,
+        files: "./dashboard/*",
```

Comment:
Actually, I'm not sure if this will work for Windows. May need to do a `path.join` here

Reply:
`files: path.join(__dirname, "dashboard"` does not work. Results in the same error. any thoughts?

Reply:
Why do we need to make a change here?  How is this related to js-yaml?

Reply:
Certain tests are hitting the following, possibly due to yaml invoking
```
[ xamples/tests/replace-secgroup ]     TypeError: Cannot read property 'map' of undefined
[ xamples/tests/replace-secgroup ]         at config.objs.then.configObjs (/opt/pulumi/node_modules/@pulumi/eks/node_modules/@pulumi/kubernetes/yaml/yaml.js:180:14
```

The theory is that this is occurring to tests that load YAML (e.g. that deploy the dashboard) with a recent change in pulumi/k8s.

See https://github.com/pulumi/pulumi-eks/issues/325 for more info